tasks/2017/intro-to-zulip-mobile.md

```diff
@@ -0,0 +1,59 @@
+# GCI Tasks: Intro to Zulip mobile development (React Native)
+
+## Prerequisites
+
+* A working Zulip mobile development environment. See
+  https://github.com/zulip/zulip-mobile/blob/master/docs/developer-guide.md#dev-environment for instructions on how to set one up.
+
+* Update your working copy of [zulip-mobile](https://github.com/zulip/zulip-mobile) and then create a feature branch. [Learn
+  how](../../before-every-task.md).
+
+### Practice Running zulip-mobile
+
+Run app using
+```
+react-native run-ios // on Mac OS, to run on iOS device (simulator or device)
+react-native run-android // to run on Android device (emulator or device)
+```
+
+If you are running on Android emulator, then make sure it is launched before you
+run command on terminal.
+
+Enter server url (we recommend to use https://chat.zulip.org) and your login credentials.
+Open left drawer and narrow to `test here` stream and send a message with your
+name as the topic. And share your experience with us in `mobile` stream with
+topic as `feedback: setting up environment | <your name>` (replace `<your name>`
+with your name).
+
+Now task is to
+* share your learning with us on https://chat.zulip.org/#narrow/stream/mobile
+* updating [docs](https://github.com/zulip/zulip-mobile/tree/master/docs)
+* take screenshot of the app.
+
+**Tips:**
+After running app, answering following questions will help you in getting ideas
+what to update in doc.
+* Does existing docs are sufficient to set up developing environment? If no,
+then what is lacking else suggest changes so that we can improve them.
+* Does you face any problem while setting up? Any error in terminal or anything
+other. Add such errors with solutions in docs.
+
+### Submit
+
+After running app,
+
+* narrow to `all private messages` from selecting envelop icon in drawer.
+Take screenshot of the simulator/emulator window.
+
+* open drawer and switch to night mode.
+Take screenshot of the simulator/emulator window.
+
+* narrow to `test here` stream, and start composing new message. Type `:smi` or
+any other emoji of your choice.
+Take screenshot of the simulator/emulator window. Make sure emoji autocomplete
+is visible in the screenshot.
+
+
+(make sure simulator/emulator window is included in the screenshots)
```

Comment:
I think you've already stated that you need to take a picture of the simulator/emulator window before :)